packages/commonwealth/client/scripts/state/ui/modals/manageCommunityStakeModal.ts

```diff
@@ -1,13 +1,14 @@
 import ChainInfo from 'client/scripts/models/ChainInfo';
-import { CommunityData } from 'client/scripts/views/pages/DirectoryPage/DirectoryPageContent';
 import { createBoundedUseStore } from 'state/ui/utils';
 import { ManageCommunityStakeModalMode } from 'views/modals/ManageCommunityStakeModal/types';
 import { devtools } from 'zustand/middleware';
 import { createStore } from 'zustand/vanilla';
 
 interface ManageCommunityStakeModalStore {
-  selectedCommunity?: ChainInfo | CommunityData;
-  setSelectedCommunity: (community: ChainInfo | CommunityData) => void;
+  selectedAddress?: string;
```

Comment:
I'd migrate to Readonly for these but it's not a blocker.  Readonly types mean no one can mangle with them and can prevent bugs.

Reply:
done